src/compas_view2/objects/nurbssurfaceobject.py

```diff
@@ -0,0 +1,48 @@
+from compas.utilities.itertools import flatten
+from .meshobject import MeshObject
+
+
+class NurbsSurfaceObject(MeshObject):
+    """Object for displaying COMPAS NurbsSurface geometry."""
+
+    def __init__(self, surface, u=10, v=10, show_edges=False, **kwargs):
+        self.u = u
+        self.v = v
+        self._mesh = surface.to_mesh(u=self.u, v=self.v)
```

Comment:
this can be done more efficiently if we directly convert the quads into triangles for visualisation, instead of first into a mesh...